packages/documentation/copy/en/developer/Downloading and Building.md

```diff
@@ -31,4 +31,4 @@ If you plan to contribute to Lingua Franca, or if you want to keep up to date as
    - Gradle: `./gradlew assemble` (the `build` also performs tests, which takes a long time)
    - Maven: `mvn compile` (you need to install Maven first)
 
-Some of code generator components are written in Kotlin, which is not supported by Eclipse. If you want a Kotlin-friendly developer environment use IntelliJ, To build the Lingua Franca IDE (Epoch) with Kotlin-based code generators enabled (which is not possible with the Eclipse setup), please see the instructions in [[Running Lingua Franca IDE (Epoch) with Kotlin based Code Generators Enabled (without Eclipse Environment)]].
+Some of code generator components are written in Kotlin, which is not supported by Eclipse. If you want a Kotlin-friendly developer environment, we recommend using IntelliJ, as described [here](https://www.lf-lang.org/docs/handbook/intellij-kotlin). <!-- To build the Lingua Franca IDE (Epoch) with Kotlin-based code generators enabled (which is not possible with the Eclipse setup), please see the instructions in [[Running Lingua Franca IDE (Epoch) with Kotlin based Code Generators Enabled (without Eclipse Environment)]]. -->
```

Comment:
I think we need to decouple the use of IntelliJ for the whole Kotlin discussion. We could start with renaming the page from `intellij-kotlin` to `intellij`.